public/plugins/ckeditor/plugins/videodetector/dialogs/videoDialog.js

```diff
@@ -0,0 +1,5 @@
+﻿CKEDITOR.dialog.add("videoDialog",function(a){return{title:"Insert a Youtube, Vimeo, Dailymotion URL or embed code",minWidth:400,minHeight:100,contents:[{id:"tab-basic",label:"Basic Settings",elements:[{type:"text",id:"url_video",label:"Youtube, Vimeo, Dailymotion URL or embed code",validate:CKEDITOR.dialog.validate.notEmpty("Empty!")}]}],onOk:function(){var b=detectar(),c="";"youtube"==b.reproductor?c="https://www.youtube.com/embed/"+b.id_video+"?autohide=1&controls=1&showinfo=0":"vimeo"==b.reproductor?
```

Comment:
This character may get silently deleted by one or more browsers.<br>Line is too long.<br>Identifier 'id_video' is not in camel case.<br>'CKEDITOR' is not defined.